game/src/main/java/game/player/PlayerCommandProcessor.java

```diff
@@ -0,0 +1,42 @@
+package game.player;
+
+import engine.command.Command;
+import game.exception.CorruptPlayerCommandException;
+import engine.exception.WrongObjectIdException;
+import engine.geometry.Direction;
+import game.command.TankMoveCommand;
+import game.object.Tank;
+
+public class PlayerCommandProcessor {
+
+    private final PlayerManager playerManager;
+
+    public PlayerCommandProcessor(PlayerManager playerManager) {
+        this.playerManager = playerManager;
+    }
+
+    public Command getCommand(PlayerCommand playerCommand) throws CorruptPlayerCommandException {
```

Comment:
refactor Command to EngineCommand to separate player control commands from commands to game engine more clearly